vendor/init.go

```diff
@@ -66,22 +83,65 @@ var files = []file{
 	},
 }
 
+func addChrome() (err error) {
+	var chromeFile file
+	chromeFile.name = chromeFilename
+	chromeFile.browser = true
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx, option.WithHTTPClient(http.DefaultClient))
+	if err != nil {
+		return fmt.Errorf("Cannot create a storage client for downloading the chrome browser: %s", err)
+	}
+	BktHandle := client.Bucket(storageBktName)
+	lcFileObj := BktHandle.Object(lastChangeFile)
+	rc, err := lcFileObj.NewReader(ctx)
+	if err != nil {
+		return fmt.Errorf("Cannot create a reader for last_change file: %s", err)
+	}
+	defer rc.Close()
+	// Read the last change file content for the latest build directory name
+	data, err := ioutil.ReadAll(rc)
+	if err != nil {
+		return fmt.Errorf("Cannot read from LAST_CHANGE: %s", err)
+	}
+	latestChromeBuild := string(data)
+	latestChromePackage := strings.Join([]string{prefixLinux64, latestChromeBuild, chromeFilename}, "/")
+	chromeObjHandler := BktHandle.Object(latestChromePackage)
+	cpAttrs, err := chromeObjHandler.Attrs(ctx)
+	if err != nil {
+		return fmt.Errorf("Cannot get the chrome package attrs: %s", err)
+	}
+	// 02x is needed as the returned []byte is not a valid utf8
+	chromeFile.hash = fmt.Sprintf("%02x", string(cpAttrs.MD5))
```

Comment:
Use `hex.EncodeToString` here since we otherwise don't need to interpolate a string.

(Also, I think `fmt.Sprintf("%x", cpAttrs.MD5)` would have been sufficient, but I think `hex.EncodeToString` is more direct).